### DISCUSSION TITLE
LND router and channel sizes

### DISCUSSION THREAD

**Discussion by GordianLN** [lightningnetwork/lnd#6560] [Q&A] [answered]
I planned my node to only have 2.1M channels, it's been a specific choice I've done vs opening fewer larger channels.
I've been reading lately though that router algos prefer larger channels because they would have higher chance to offer liquidity where it's needed.
Leaving out other LN implementations, and custom routers, does LND do this?
My node is completely automated, and quite efficient at this, and such policy would strip smaller nodes of their usefulness in the network.

**Comment by guggero** [comment, marked as answer]
The discussion here might be relevant: https://github.com/lightningnetwork/lnd/issues/5988

**Reply by GordianLN** [reply]
Thank you for your reply!
Yes, I was pointed to that convo a while ago, and I am not going to post in there, as that thread is full of giants and I don't want to get stomped upon.
My observation is that the logic explained in there is sound, and not only that, I would code a router MC just like that, myself.

In other words, if channel sizes are taken into account only relative to HTLC size, then I, who have a trademark capacity of 2.1M, am totally fine only routing up to 4-500k payments, make it even 2-300k... *as long as* this doesn't mean that, by having channels that are about 50% of the average channel size as reported by amboss, I tend to be ignored altogether by other lnd nodes because "2.1M is too small", no matter the HTLC size.
